src/main/java/mate/academy/service/UserService.java

```diff
@@ -14,7 +14,15 @@
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
-    public User findByEmail(String email) {
-        return null;
+    public static User findByEmail(String email) {
+        User l = null;
+        if (email.matches("[A-Za-z].*?@i\\.ua")) {
```

Comment:
to avoid the hardcoding you can create static final field (f.e. `EMAIL_MATCHER`) and assign it by your regex "[A-Za-z].*?@i\\.ua". Than you can pass this field to `matches()`